controllers/articles.controller.js

```
@@ -19,4 +19,22 @@ exports.getArticles = (req, res, next) => {
     .catch((err)=>{
         next(err);
     })
+}
+
+// patch
+exports.patchArticle = (req, res, next) => {
+    if(isNaN(req.body.inc_votes)) {
+        next({errorCode: 400, errorMessage: 'Invalid body'})
+    }
```

PSQL should throw an error here you could use

---

i moved this check to model